ytools/y2tool/rubocop_yast_style.yml

```diff
@@ -199,3 +202,46 @@ Style/AsciiComments:
 #
 Style/DoubleNegation:
   Enabled: false
+
+# New Rubocop uses a different default ("special_inside_parentheses")
+# which adds too much indentation in some cases. Use the backward compatible value.
+Style/IndentArray:
+  EnforcedStyle: consistent
+
+# Use the backward compatible default
+Lint/EndAlignment:
+  AlignWith: start_of_line
+
+#  Allow block.call:
+#
+#  def foo(&block)
+#    var = block,call
+#    ...
+#
+# is better readable than
+#
+#  def foo
+#    var = yield
+#    ...
+#
+#  athough it is a bit slower
+Performance/RedundantBlockCall:
+  Enabled: false
+
+# Use the backward compatible default
+Style/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+# Do not check the module length, the YCP converted includes
+# are usually too long.
+Metrics/ModuleLength:
+  Enabled: false
+
+# Disable the "foo.size == 0" check, Yast::Term does not provide #empty?
```

Comment:
Um, I saw [this comment](https://github.com/yast/yast-yast2/blob/8eb9214265fdf8afcae1531c5a0309799028d79d/library/packages/src/modules/PackagesUI.rb#L317) and disabled it without verifying that...

I'll enable it.